services/web-app/pages/about-carbon/community-sites.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Community sites
+description:
+  These sites within the Carbon community include guidance, assets, and resources for all of IBM's
+  digital experiences from product to website. Some guidance is accessible to IBMers only.
+---
+
+<PageDescription>
+
+These sites within the Carbon community include guidance, assets, and resources for all of IBM's
+digital experiences from product to website. Some guidance is accessible to IBMers only.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Product</AnchorLink>
+  <AnchorLink>IBM.com</AnchorLink>
+  <AnchorLink>Brand</AnchorLink>
+  <AnchorLink>Conversational UX</AnchorLink>
+</AnchorLinks>
+
+## Product
+
+The guidance and resources here are for IBM product teams building software applications and digital
+experiences with the Carbon Design System. This work is built on the foundation of Carbon and
+provides solutions for specific domains and use cases.
+
+_This content is accessible to IBMers only._
+
+<Row className="resource-card-group">
```

Comment:
```suggestion
<CardGroup>
```

Looks like we have a bug where the resource-card-group class doesn't wrap correctly past one row, easiest fix is to switch this to the CardGroup component.

Reply:
Thanks, will do. I copy / pasted all MDX, and just updated image paths to images I added to the `/public` directory.

Reply:
Done.